scripts/process-data/fitbitecg.R

```diff
@@ -0,0 +1,212 @@
+# Functions
+
+#' Create i2b2 summaries
+#'
+#' `ecg_stat_summarize()` summarizes the ECG data in a data frame on 
+#' specific time scales (all data and weekly) for the following statistics: 
+#' Number of records.
+#'
+#' @param df A data frame with `participantidentifier`, `concept`, `value`, and 
+#' some combination of `startdate`, `enddate`, `date`, or `datetime` columns.
+#'
+#' @return A data frame that differs in size from `df`. The original `value` 
+#' column is dropped and replaced by a new `value` column containing the values 
+#' of each summary statistic computed for each group of data (group by 
+#' `participantidentifier` and `concept` for all data, and 
+#' `participantidentifier`, `concept`, `year`, `week` for weekly summaries).
+ecg_stat_summarize <- function(df) {
```

Comment:
I'm wondering why you've written a standalone function here, instead of writing a general purpose function for categorical variables in RecoverSummarizeR? For an initial commit, I think this is fine, but for utility going forward it might be worth making the change in the future..

Reply:
I haven't gotten around to updating the `recoverSummarizeR` package yet. Among adding this function for categorical variables, there are a couple of package-wide changes I plan on making, so I will add the function in that major release and then update this script accordingly.